setup-artifactory-service-account-repository-project.md

```diff
@@ -0,0 +1,163 @@
+
+The project is also given a shortname, called a project key. This is created automatically using the first letter of your `APname` and three digits of your namespace name. Using the previous example, the project key is `ta1b`. The project key must be unique.
+
+When Archeobot sees a new ArtifactoryProject in your namespace, it sends the Platform Team a note that you want a new Project, which the team must approve. While you wait for your new ArtifactoryProject to be approved, you can look at it using the following command:
+
+`oc describe artproj [APname]`.
+
+In `spec`, there's an entry for your project key and an `approval_status` box that should show **pending**. This means that we haven't approved or rejected your request for an Artifactory project.
+
+When the project is approved, the `approval_status` box should show **approved**. In this case, Archeobot knows that your ArtifactoryProject is approved, but it hasn't yet made the Project for you in Artifactory.
+
+You may also see **nothing-to-approve**. This status is meant to say that there are no outstanding changes to be made. If you see this, it means that your project is created in Artifactory, and you can [sign in](https://artifacts.developer.gov.bc.ca).
+
+**Note**: While you are able to patch the `approval_status` box, it doesn't mean you can approve your own project. The box is there for informational purposes. If you change it, Archeobot changes it back.
+
+### Rejected requests for projects or quota increases
+If you've made a request for a new Artifactory project or quota increase and it was rejected, reach out to the Platform Services team to ask why. If you think your request was rejected in error, let the team know. If the request should have been approved, they'll switch the status to `approved` and you'll see the changes applied.
+
+If the team maintains the rejection, make sure you acknowledge the rejection. If you don't, further change requests are ignored. Acknowledge the rejection by changing your Artifactory projectProject object back to the state it was in before you made the request.
+* If you requested a new project, delete the ArtifactoryProject object from your namespace.
+* If you requested an increased quota, change the quota in the spec of your ArtifactoryProject back to the currentquota of your project. If you're not sure what the quota is, check the Overview page for your project in the Artifactory web GUI.
+
+After you've made the change, Archeobot reconciles once more and you'll see that your `approval_status` changes back to `nothing-to-approve` (or you will have deleted the ArtifactoryProject object, in which case you won't see anything at all). This means your rejection has been acknowledged and you can make further change requests.
```

Comment:
```suggestion
```

This section should be deleted since I moved all of it up above the other stuff.